product_docs/docs/migration_portal/4/04_mp_migrating_database/02_mp_schema_assessment.mdx

```diff
@@ -109,15 +109,17 @@ An object is compatible if it successfully executes on a target database. In the
 
 ### Feature compatibility
 
-Some features might not be natively available in EDB Postgres Advanced Server. The report presents a summary of feature occurrences and compatibility levels.
+Some features might not be natively available in EDB Postgres Advanced Server. The assessment report presents a summary of feature occurrences and compatibility levels.
 
 | Feature compatibility level                                    | Workaround status                                                                                                                                               |
 |----------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | Compatible features                                            | -                                                                                                                                                               |
 | Features with existing available workaround                    | Exists                                                                                                                                                          |
 | Features with reasonable effort new workaround                 | Possible                                                                                                                                                        |
-| Features with no workaround or requiring further investigation | None / NeedsInvestigation. For features with no workaround, it might be possible to address the feature incompatibility through application-level changes. |
+| Features with no workaround or requiring further investigation | None / Needs Investigation /* |
 
+ /* For features with no workaround, it might be possible to address the feature incompatibility through application-level changes.
```

Comment:
Will the /* render as * or as /*?

Reply:
Based on the deployment that @drothery-edb just generated, "/*" is being rendered as is (i.e., "/*").  Was this meant to render as "*" instead?

Reply:
@moiznalwalla to escape the * character, you need to use the backslash (\) not the forward slash (/).